### DISCUSSION TITLE
Installing on Apple silicon m1/m2

### DISCUSSION THREAD

**Discussion by Vrk3ds** [hlky/stable-diffusion-webui#803] [Q&A]
I am a complete novice to this but was able to get the lstein fork running on my Mac Studio m1. It’s great but I would love to have access to the gradio ui. Has anyone been able to get this running on Apple silicon m1/m2?

**Comment by milezzz** [comment]
Same question! Someone throw us a lifeline please, I see conflicting reports on whether it works or not.

**Comment by rudolfolah** [comment]
There was a thread in another repo with suggestions on how to get the webui running on an Apple Silicon M1, however it seems to be gone.

Most recently, there's a one-click install app that works and seems to run quickly: https://github.com/divamgupta/diffusionbee-stable-diffusion-ui

The requirements.txt is a bit different from the `environments.yaml` that is in this repo: https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/blob/master/stable-diffusion/requirements.txt#L11
